test/bats.bats

```diff
@@ -221,11 +221,11 @@ fixtures bats
   [ $status -eq 0 ]
 }
 
-@test "load supports scripts on the PATH" {
+@test "load supports scripts on the BATS_LIB_PATH" {
```

Comment:
No, loading scripts from PATH was never my intention and I've stated that in both the original PR and the comment the the recent rebase.
If the user wants to load scripts from PATH they can add it to BATS_LIB_PATH if they desire.

Reply:
Ah, that wasn't clear to me from `The existing functionality of load() is preserved` in this PR. 

I can't say I use `PATH` to load libraries so didn't pick it up, so to be clear: this is an intentionally breaking change for users that do use PATH?

Reply:
Well, PATH wasn't in the picture for `load` before the last release.

If you want to maintain that `PATH` is targeted by default that can be added. But I'd advice strongly against that, I've noted previously that this could cause confusion when a BATS library is created for an existing application.
The example I've noted previously is `xvfb-run`, where both the BATS library and the application itself would be called `xvfb-run`, confusing `load`.

That `PATH` can be added to `BATS_LIB_PATH` can of course be documented in the readme's `load` section.

Reply:
It is really unfortunate, that we got into this situation because this solution is much cleaner. Nevertheless, we have to deal with the fact that some users might rely on PATH now. I added a comment for this in my review so we can continue the discussion over there. The gist is: We should preserve the current behavior but transition to the suggestion by @ntnn without hard breaks for existing users.